### DISCUSSION TITLE
"file /proc/acpi/call does not exist" Error

### DISCUSSION THREAD

**Discussion by SunkenHero** [dominiksalvet/asus-fan-control#119] [Q&A]
So i installed AFC on my Asus laptop (TP470EA), but i am getting the error msg "file /proc/acpi/call does not exist". I am sure its easy to fix, but I am not that expirienced to do that. So mayby someone wants to help me. I got acpi and acpi-call-dkms installed.

**Comment by dominiksalvet** [comment]
Maybe you don't have the module loaded. Try `modprobe acpi_call` and repeat. This should be done automatically by the systemd service.

**Reply by SunkenHero** [reply]
So if i type it into the terminal with sudo i get no output. I would assume, that the module isn't loaded, but i don't know how to load it.

**Reply by dominiksalvet** [reply]
Try loading module as described and then `sudo asus-fan-control`. It should work considering the module was loaded successfully.

**Reply by SunkenHero** [reply]
```
tobi@tobi-laptop:~$ sudo modprobe acpi_call
tobi@tobi-laptop:~$ sudo asus-fan-control 
acpi call failed with 'Error: AE_NOT_FOUND'
tobi@tobi-laptop:~$ 
```
Ok its still pulling an error. Did i do something wrong?

**Reply by dominiksalvet** [reply]
I don't think you did something wrong. You probably have an incompatible device. Have a look at some issues with the AE_NOT_FOUND label. Unfortunately, that may be a dead end...
